src/doc/DocHighlightThread.js

```diff
@@ -72,6 +71,8 @@ class DocHighlightThread extends AnnotationThread {
      */
     destroy() {
         this.threadID = null;
+
+        // $FlowFixMe
         this.emit(THREAD_EVENT.render, this.location.page);
```

Comment:
Guard for `this.location.page` since `location` can now be null/undefined?

Reply:
Handled it in the method that handles this event :)